Dockerfile

```diff
@@ -1,12 +1,23 @@
 # Read the Docs - Environment base
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 MAINTAINER Read the Docs <support@readthedocs.com>
 LABEL version="latest"
 
 ENV DEBIAN_FRONTEND noninteractive
 ENV APPDIR /app
 ENV LANG C.UTF-8
-ENV CONDA_VERSION 4.3.31
+
+# Versions, and expose labels for exernal usage
+ENV PYTHON_VERSION_27 2.7.14
+ENV PYTHON_VERSION_33 3.3.7
```

Comment:
Do we want to support 3.3 but not 3.4? I think both are not heavily used.

Reply:
Yeah, i meant to ask. I'm 👍 on removing 3.3